### PR TITLE
Visit scheduler changes

### PR DESCRIPTION
**Pull request by hutcheonb-moj**
## Description
Imported visit scheduler changes
biWeekly -> weekly frequency
ACTIVE_OR_FUTURE -> CURRENT_OR_FUTURE (session template filter)

*(Can't run integration tests locally so can't work out what the reported problem is)*

### CODE REVIEW

**Comment by tpmcgowan** [comment]
Looks good. Haven't checked, but I suspect the integration test issue is this line: https://github.com/ministryofjustice/hmpps-visits-internal-admin-ui/blob/4ba455d87283165aa6f30afe1a072c59b78d15eb/integration_tests/mockApis/visitScheduler.ts#L87

The wiremock stub just uses the default filter option (which has changed).

**Review by AledWEvans — APPROVED** [review]
LGTM, some test failing though :
`ACTIVE_OR_FUTURE` should now be changed to `CURRENT_OR_FUTURE`

```
  stubGetSessionTemplates: (prisonCode: string): SuperAgentRequest => {
    return stubFor({
      request: {
        method: 'GET',
        url: `/visitScheduler/admin/session-templates?prisonCode=${prisonCode}&rangeType=CURRENT_OR_FUTURE`,
```
Surprised that we are not using the type (SessionTemplatesRangeType) directly here, therefore you would have compile error...

File: integration_tests/mockApis/visitScheduler.ts [hunk 1]
```diff
@@ -80,11 +81,17 @@ export default {
...
+  }: {
+    prisonCode: string
+    rangeType: SessionTemplatesRangeType
+  }): SuperAgentRequest => {
     return stubFor({
       request: {
         method: 'GET',
-        url: `/visitScheduler/admin/session-templates?prisonCode=${prisonCode}&rangeType=ACTIVE_OR_FUTURE`,
+        url: `/visitScheduler/admin/session-templates?prisonCode=${prisonCode}&rangeType=${rangeType}`,
       },
```
Comment (AledWEvans) [inline integration_tests/mockApis/visitScheduler.ts:95]: Very nice

**Review by AledWEvans — APPROVED** [review]
LGTM

**Review by tpmcgowan — APPROVED** [review]
LGTM

**Pull request merged**